code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -141,13 +141,11 @@
 		return occupant
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/container_resist(mob/user)
-	usr << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds and you need to stand still.)</span>"
+	user << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds and you need to stand still.)</span>"

```

Comment:
But how does the player know that the resist process will free him in 30 seconds if we don't tell him?